docs/guides/operator-guide-aws-marketplace.md

```diff
@@ -350,6 +350,23 @@ The following table includes some additional commands that might help you manage
 | Runs one iteration of `logrotate` manually, without changing the scheduled interval. | `sudo logrotate -f /etc/logrotate.conf --force` |
 | Reloads `syslog-ng`. | `sudo /usr/sbin/syslog-ng-ctl reload` |
 
+### UID2 Operator error codes
+
+The following table lists possible errors while setting up private operator.
+
+:::note
+This applies only to Private Operators starting from Q2 2024.
+:::
+
+| Error Code | Issue | Steps to Resolve |
+| :--- | :--- | :--- |
+| AW01 | MissingInstanceProfile |  Attach an IAM instance profile to the EC2 instance with the required permissions. The UID2 Operator needs these permissions to access configurations from AWS Secrets Manager. |
+| AW02 | ConfigNotFound | Ensure the secret referenced by the operator exists in AWS Secrets Manager in the same region as the operator. And verify that the IAM instance profile has permission to access it. Check the logs the specific secret name and region. |
```

Comment:
Edits for column 3:

| AW02 | ConfigNotFound | Take the following steps:<br/>- Ensure the secret referenced by the operator exists in AWS Secrets Manager in the same region as the operator.<br/>- Verify that the IAM instance profile has permission to access the secret.<br/>- Check the logs for the specific secret name and region. |

Not totally sure if there's a condition on step 3... or it's really part of step 2... they check the logs, and then what? Or is this a way of making sure it's now working? Not sure.